project-base/storefront/components/Forms/Select/Select.tsx

```diff
@@ -45,6 +45,12 @@ const Control = (props: any) => {
 };
 
 export const Select: FC<SelectProps> = ({ hasError, onChange, options, defaultValue, isDisabled, value, ...props }) => {
+    const [isClient, setIsClient] = useState(false);
+
+    useEffect(() => {
+        setIsClient(true);
+    }, []);
```

Comment:
Couldn't we use `isClient` from utils?

Reply:
Ah, cool. I haven't used it yet. Thanks!